.github/workflows/helm-chart-lint.yaml

```diff
@@ -1,18 +1,21 @@
 name: Lint and Test Charts
 
-# Run chart linting and tests on each pull request
 on:
```

Comment:
as the comment from @nicoprow you can leave this trigger as it is... the workflow dispatch option is just another trigger to the workdlow...

Reply:
As discussed before we align right now more with the example in the TRGs and from then on make some adaptations if necessary